src/coverlet.core/Helpers/InstrumentationHelper.cs

```diff
@@ -420,7 +420,7 @@ public bool IsTypeIncluded(string module, string type, string[] includeFilters)
     }
 
     public bool IsLocalMethod(string method)
-        => new Regex(WildcardToRegex("<*>*__*|*"), s_regexOptions, TimeSpan.FromSeconds(10)).IsMatch(method);
+        => new Regex(WildcardToRegex("<*>*__*|*")).IsMatch(method);
```

Comment:
I would make this regex a static instance, there's no reason to keep creating it.

Reply:
The method `IsLocalMethod` is used in a loop which examines all `method.Name` and therefore this is created one time.

This PR resolves the performance issue and we should limit the bug fix scope.

Reply:
It will be created every time IsLocalMethod is called in the loop, no?